ui/src/common/layouts/CollectionsMenu/CollectionsMenu.jsx

```diff
@@ -68,12 +70,35 @@ function CollectionsMenu({ currentPathname }) {
         </CollectionLink>
       </Col>
       <Col>
-        <CollectionLink
-          active={activeCollection === INSTITUTIONS_PID_TYPE}
-          to={INSTITUTIONS}
+        <Menu
+          className="__CollectionsDropdownMenu__ mh4 m-mh2"
+          theme="dark"
+          mode="horizontal"
+          selectable={false}
+          inlineIndent={0}
         >
-          Institutions
-        </CollectionLink>
+          <Menu.SubMenu key="More" title="More...">
```

Comment:
You should use `DropdownMenu` instead, since `SubMenu` has some features we don't need here.